src/client/components/Results.jsx

```diff
@@ -141,13 +141,13 @@ export default class Results extends Component {
 
         topMessage = (
           <div className="result-summary">
-            <Pluralize singular="distinct message" count={items.length} /> (of which:{ofWhich}) [<Pluralize singular="message" count={this.props.results.length} /> in total returned]
+            <Pluralize singular="distinct message" count={items.length} /> (of which:{ofWhich}) [<Pluralize singular="message" count={this.props.results.length} /> in total returned from the validator in "{this.props.validationMode}" mode]
```

Comment:
The bracket is in slightly the wrong place here.

The code here would render:
`11 distinct messages (of which: 2 Notices, 7 Warnings, 2 Tips) [16 messages in total returned in "C1 Response" mode]`

However it should render:
`11 distinct messages (of which: 2 Notices, 7 Warnings, 2 Tips) [16 messages in total returned] in "C1 Response" mode`

Or actually even better:
`11 distinct messages (of which: 2 Notices, 7 Warnings, 2 Tips) [16 messages in total] returned from the validator in "C1 Response" mode`

Reply:
Agreed, last one is much more consistent - changed to match. It now looks like:
`30 distinct messages (of which: 6 Errors, 19 Warnings, 5 Tips) [31 messages in total] returned from the validator in "C2Response" mode`

Reply:
Great!